src/workflows-app/components/submission-common.js

```diff
@@ -103,6 +104,70 @@ export const parseMethodString = (methodString) => {
   };
 };
 
+const inputButtonRowStyle = {
+  height: '2.5rem',
+  width: '100%',
+  display: 'flex',
+  alignItems: 'center',
+};
+
+export const InputsButtonRow = ({
```

Comment:
small nit: I am wondering that since this is not a utility function maybe this could be made a component and put inside `../src/workflows-app/components` folder?

Reply:
I notice there's some other components in this file as well. What if we make this file component-only and move the utility functions to a `../src/workflows-app/utils/submission-utils.js`?